x-pack/plugins/licensing/public/license.test.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ILicense } from '../common/types';
+import { LICENSE_STATUS } from '../common/constants';
+import { LicenseFeature } from '../common/license_feature';
+import { setup } from './__fixtures__/setup';
+import { Plugin } from './plugin';
+
+describe('license', () => {
+  let plugin: Plugin;
+  let license: ILicense;
+
+  afterEach(async () => {
+    await plugin.stop();
+    sessionStorage.clear();
+  });
+
+  test('uid returns a UID field', async () => {
+    ({ plugin, license } = await setup());
```

Comment:
why we couple License tests to Plugin interface?

Reply:
All tests work this way, we need the dependencies of the item in order to test it. This also helps keep the tests smaller.